src/Aspire.Hosting.Azure/AzurePublisher.cs

```diff
@@ -13,6 +14,7 @@
 
 namespace Aspire.Hosting.Azure;
 
+[Experimental("ASPIREAZURE001", UrlFormat = "https://aka.ms/dotnet/aspire/diagnostics#{0}")]
```

Comment:
I know this is just following precedent of other experimental attributes, but in hindsight, I wonder if it would've been better to just have these attributes be all ASPIRE0xxx and just increment the number. That seems to be the pattern that some of the other repos using experimental do anyway.

Reply:
@mitchdenny - thoughts on this? I think you set the precedent as you were the first to make experimental APIs.

https://github.com/dotnet/aspire/pull/3383/commits/9f499dbe81351c3c0f2598f7e0d294254555fe57